rules/scala/workspace.bzl

```diff
@@ -21,12 +21,12 @@ def scala_artifacts():
         "org.scala-sbt:util-interface:1.3.0",
         "org.scala-sbt:util-logging_2.12:1.3.0",
         "org.scala-sbt:compiler-interface:1.3.4",
-        "org.scala-sbt:zinc-compile-core_2.12:1.3.4",
-        "org.scala-sbt:zinc_2.12:1.3.4",
-        "org.scala-sbt:zinc-persist_2.12:1.3.4",
-        "org.scala-sbt:zinc-core_2.12:1.3.4",
-        "org.scala-sbt:zinc-apiinfo_2.12:1.3.4",
-        "org.scala-sbt:zinc-classpath_2.12:1.3.4",
+        "org.scala-sbt:zinc-compile-core_2.12:1.4.0",
+        "org.scala-sbt:zinc_2.12:1.4.0",
+        "org.scala-sbt:zinc-persist_2.12:1.4.0",
+        "org.scala-sbt:zinc-core_2.12:1.4.0",
+        "org.scala-sbt:zinc-apiinfo_2.12:1.4.0",
+        "org.scala-sbt:zinc-classpath_2.12:1.4.0",
         "ch.epfl.scala:bloop-frontend_2.12:1.0.0",
         "com.thesamet.scalapb:scalapb-runtime_2.12:0.9.0",
     ]
```

Comment:
We also need to fetch the correct compiler bridge artifacts below (for `compiler_bridge_2_11` and `compiler_bridge_2_12`)

Reply:
Fixed this